package.json

```diff
@@ -47,65 +47,69 @@
     "@graphql-codegen/typescript": "^1.23.0",
     "@graphql-codegen/typescript-document-nodes": "^1.17.16",
     "@graphql-codegen/typescript-operations": "^1.18.4",
-    "@storybook/addon-a11y": "^6.3.2",
-    "@storybook/addon-actions": "^6.3.2",
-    "@storybook/addon-controls": "^6.3.2",
-    "@storybook/addon-docs": "^6.3.2",
-    "@storybook/addon-measure": "^2.0.0",
-    "@storybook/addon-storyshots": "^6.3.2",
-    "@storybook/addon-storysource": "^6.3.2",
-    "@storybook/addon-viewport": "^6.3.2",
-    "@storybook/addons": "^6.3.2",
-    "@storybook/client-api": "^6.3.2",
-    "@storybook/react": "^6.3.2",
+    "@storybook/addon-a11y": "^6.5.0-alpha.39",
+    "@storybook/addon-actions": "^6.5.0-alpha.39",
+    "@storybook/addon-controls": "^6.5.0-alpha.39",
+    "@storybook/addon-docs": "^6.5.0-alpha.39",
+    "@storybook/addon-measure": "^6.5.0-alpha.39",
```

Comment:
there's a bug in that the storybook folks are addressing. In the meantime we need to use this alpha release